### ISSUE TITLE
Is this still working?

### ISSUE THREAD

**Issue by xetra11** [jfim/org-jira#2]
Hey just wanted to check the status of this because I really would love to have it.
And what about spacemacs integration?

**Comment by nickanderson** [comment]
Yeah, do you plan to open a PR to add the layer upstream? Plan to give this a try soon. really hopeful!

**Comment by jfim** [comment]
Sorry, I've been pretty busy with work. I'll see if I can make an upstream PR this weekend.

**Comment by xetra11** [comment]
@jfim 
would appreciate it!

**Comment by nickanderson** [comment]
I just gave it a go, I am on the develop branch.

Good instructions, but when i try to get issues with `, j i g` I get `error retrieving WSDL 404.` A quick search turns up an old issue https://github.com/baohaojun/org-jira/issues/2 which mentions a restapi branch.

**Comment by jfim** [comment]
The `master` branch has been updated to use the REST API instead of the deprecated SOAP API, so the instructions in the readme should work.

**Comment by sehqlr** [comment]
BTW, I recently starting using this, and the instructions in the README are still current, even after several months, so @jfim if you wanted confirmation that it works, you have it from me.

**Comment by jfim** [comment]
Thanks @sehqlr!
